### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by monperrus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,11 @@
 
 ## Getting Started
 
-To build it, we use [mill](http://www.lihaoyi.com/mill/). I you want to run the specification tests, just type:
+To build it, we use [mill](http://www.lihaoyi.com/mill/). Mill is not available on on Unix boxes, but you can you use curl to install it (see Installation instructions in the [doc](http://www.lihaoyi.com/mill/)):
```
Comment (satabin) [inline README.md]:
```suggestion
To build it, we use [mill](http://www.lihaoyi.com/mill/). Mill is not packaged all Linux distributions, but you can you use curl to install it (see Installation instructions in the [doc](http://www.lihaoyi.com/mill/)):
```

Technically, it is available on some Unixes, e.g. MacOS through brew and Arch Linux in AUR, and FreeBSD has a port. Main lack is for Debian based and Redhat based distros :)

File: README.md [hunk 2]
```diff
@@ -8,7 +8,11 @@
...
+
+    sh -c '(echo "#!/usr/bin/env sh" && curl -L https://github.com/lihaoyi/mill/releases/download/0.3.6/0.3.6) > $HOME/bin/mill && chmod +x $HOME/bin/mill'
```
Comment (satabin) [inline README.md]: I will have to think about upgrading this line when I upgrade mill :(

File: README.md [hunk 3]
```diff
@@ -8,7 +8,11 @@
...
+
+I you want to run the specification tests, just type:
 
     $ mill runtime.test
```
Comment (satabin) [inline README.md]: I didn't realized you used that syntax, can you use the fenced code block instead please (for consistency across all markdwon files)

File: README.md (hunk 2 above)
Comment (monperrus) [reply README.md]: agree. Unfortunately, github does not support `https://github.com/lihaoyi/mill/releases/download/latest/latest`

File: README.md (hunk 3 above)
Comment (monperrus) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (satabin) [reply README.md]: I actually doubt this makes sense here, because, it is simply copy/paste of what is in the mill documentation.

Comment (monperrus) [reply README.md]: I like self-contained getting started without opening any external URL.

Do you want me to remove this?

Comment (satabin) [reply README.md]: Well, it is not really, as OSX, Arch Linux, and FreeBSD users don't have the preferred way of doing it here, so this looks like this is the only way to isntall mill (which it is not). I would remove it. Your readme can never be self contained, unless it integrates the documentation for every used tool. And I don't want to do that.

Comment (monperrus) [reply README.md]: no need to further discuss. removed.

**Pull request merged**